src/many-types/src/blockchain.rs

```diff
@@ -83,8 +83,11 @@ pub struct Transaction {
     #[n(0)]
     pub id: TransactionIdentifier,
 
-    #[cbor(n(1), with = "minicbor::bytes")]
-    pub content: Option<Vec<u8>>,
+    #[n(2)]
```

Comment:
See https://github.com/many-protocol/specification/pull/83 for an explanation of this index shift.